pom.xml

```diff
@@ -136,6 +137,11 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>net.spals.midas</groupId>
```

Comment:
How do you set it up such that these dependencies don't automatically import to child dependencies? And so the children have to explicitly include what dependencies from the parent they won't but they don't require the explicit version?